signin.js

```diff
@@ -0,0 +1,116 @@
+const emailInput = document.querySelector('.email-input');
+const pwdInput = document.querySelector('.pwd-input');
+const loginButton = document.querySelector('.button-sign');
+const checkPwd = document.querySelector('.check-pwd');
+const eyeIcon = document.querySelector('.pwd-eye-off');
+const emailError = document.querySelector('.email-error');
+const pwdError = document.querySelector('.pwd-error');
+const eyeOnoff = document.getElementById("eyeOnOff");
+const input = document.querySelectorAll('input');
+const VALID_EMAIL = 'test@codeit.com';
+const VALID_PASSWORD = 'codeit101';
+
+function checkEmailValid(emailAddress) {		
+    const email_regex = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/i;
+    return email_regex.test(emailAddress);
+}
+
+function addEmailErrorSign() {
+    emailError.classList.remove('hide');
+    emailInput.classList.add('error-border');
+}
+
+function removeEmailErrorSign() {
+    emailError.classList.add('hide');
+    emailInput.classList.remove('error-border');
+}
+
+function addPwdErrorSign() {
+    pwdError.classList.remove('hide');
+    pwdInput.classList.add('error-border');
+}
+
+function removePwdErrorSign() {
+    pwdError.classList.add('hide');
+    pwdInput.classList.remove('error-border');
+}
+
+function loginCheck() {
+    if (emailInput.value === VALID_EMAIL && pwdInput.value === VALID_PASSWORD) {
+        location.href = 'folder.html';
+    } else {
+        addEmailErrorSign();
+        emailError.innerText = '이메일을 확인해주세요';
+        addPwdErrorSign();
+        pwdError.innerText = '비밀번호를 확인해주세요';
+    }
+};
+
+emailInput.addEventListener('focusout', () => {
+    if (emailInput.value === '') {
+        emailError.innerText = '이메일을 입력해주세요';
+        addEmailErrorSign();
+    } 
+    else {
```

Comment:
else 는 앞에 중괄호랑 한 줄로 써야해요! 
동일하게 작성된 다른 부분들도 확인해주세요~

```suggestion
    } else {
```